documentation/modules/rn-2.6.adoc

```diff
@@ -58,9 +58,25 @@ The {project-short} web console can no longer download logs. With this update, y
 
 This part provides features and enhancements introduced in {project-full} 2.6:
 
+.Preservation of a static IP in a migration from vSphere
+
+You can now preserve a static IP when importing VMs that have vSphere as their source provider. link:https://issuesredhat.com/browse/MTV-1148[(MTV-1148)]
```

Comment:
Broken link 

```suggestion
You can now preserve a static IP when importing VMs that have vSphere as their source provider. link:https://issues.redhat.com/browse/MTV-1148[(MTV-1148)]
```

Reply:
as it can be more than a single IP and it's limited to Windows, how about:
```suggestion
You can now preserve static IPs of VMs that are installed with Windows operating system and are migrated from vSphere. link:https://issues.redhat.com/browse/MTV-1148[(MTV-1148)]
```